databases/week3/mealsharing_database.sql

```diff
@@ -0,0 +1,36 @@
+DROP  DATABASE mealSharing;
+CREATE  DATABASE mealSharing;
+USE  mealSharing;
+
+CREATE TABLE `Meal` (
+  `id` int  unsigned auto_increment  primary key ,
+  `title` varchar(255) NOT NULL ,
+  `description` text NOT NULL ,
+  `location` varchar(255) NOT NULL ,
+  `time` datetime NOT NULL ,
+  `max_reservations` int NOT NULL ,
+  `price` decimal ,          
+  `created_date` datetime NOT NULL default CURRENT_TIMESTAMP()
+);
```

Comment:
You forgot to define the character set and the engine in all the tables, see here for an example: https://github.com/HackYourFuture-CPH/databases/blob/master/lesson1/lesson-data.sql